.travis.yml

```diff
@@ -72,10 +72,6 @@ script:
     - travis_wait bash composer install -o  --no-interaction --no-progress --prefer-dist
     - make prepare DB=$DB
 
-    - echo "travis_fold:start:migrations"
```

Comment:
`travis_fold` is really cool to avoid noise lines on travis log.
Maybe you can put them around make prepare?